src/coreclr/vm/exceptionhandling.h

```diff
@@ -384,6 +384,7 @@ class ExceptionTracker
                     X86_ARG(PT_CONTEXT pContextRecord)
                     ARM_ARG(PT_CONTEXT pContextRecord)
                     ARM64_ARG(PT_CONTEXT pContextRecord)
+                    LOONGARCH64_ARG(PT_CONTEXT pContextRecord)
```

Comment:
I am deleting this platform-specific arg in #65624

Reply:
OK, Thanks